config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
   resources 'messages', :only => [:index, :new, :create, :show, :destroy]
+  
+  get 'recipients/:id' => 'recipients#show'
```

Comment:
rails can generate RESTful routes automagically for you.  Here you would use:

``` ruby
resources :recipients, only: [:show]
```

and if you wanted to use an unconventional route, you would do:

``` ruby
resources :recipients, only: [:show] do
  get "authenticated" on: :member # gets with an id e.g. recipients/:id/authenticated
  get "authenticated" on: :collection # gets a collection e.g. recipients/authenticated
end
```